sidebars.js

```diff
@@ -38,6 +38,13 @@ module.exports = {
               }
             ]
           },
+          {
+            type: 'category',
```

Comment:
Do you think we will need more articles on this in the future? If not I think we don't need another category for it.

Reply:
Will fix it

Reply:
done